src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -470,15 +470,16 @@ function _createGeckoThread(extraMarkers = []): GeckoThread {
         stack: 0,
         time: 1,
         eventDelay: 2,
+        threadCPUDelta: 3,
```

Comment:
don't you need to add a `sampleUnits` property in `meta` too, so that the profile is consistent ?

Reply:
Oh, I see you added it in the next commit. Maybe it makes sense to merge the 2 commits, because they're related. Up to you!